App/Containers/ThreadsList.js

```diff
@@ -138,6 +153,50 @@ class ThreadsList extends React.PureComponent {
     )
   }
 
+  _renderList () {
+    if (
+      this.props.threads &&
+      this.props.threads.length !== 0 &&
+      this.props.tourScreen !== true
+      ) {
+      return (
+        // FIXME: This should be a FlatList for sure
+        <View style={styles.contentContainer} >
+          <FlatList
+            data={this.props.threads}
+            keyExtractor={this._keyExtractor}
+            renderItem={this._renderItem}
+            refreshing={this.props.refreshing}
+            onRefresh={this._onRefresh}
+            initialNumToRender={4}
+          />
+        </View>
+      )
+    }
+    return null
+  }
+
+  _renderPlaceholder () {
```

Comment:
Only render placeholder if we either have an empty threads object, or the threads array is empty.